modules/cluster-logging-deploy-console.adoc

```diff
@@ -235,7 +235,7 @@ However, an unmanaged deployment does not receive updates until OpenShift Loggin
 [NOTE]
 +
 ====
-The maximum number of Elasticsearch master nodes is three. If you specify a `nodeCount` greater than `3`, {product-title} creates three Elasticsearch nodes that are Master-eligible nodes, with the master, client, and data roles. The additional Elasticsearch nodes are created as Data-only nodes, using client and data roles. Master nodes perform cluster-wide actions such as creating or deleting an index, shard allocation, and tracking nodes. Data nodes hold the shards and perform data-related operations such as CRUD, search, and aggregations. Data-related operations are I/O-, memory-, and CPU-intensive. It is important to monitor these resources and to add more Data nodes if the current nodes are overloaded.
+The maximum number of Elasticsearch control plane nodes (also known as the master nodes) is three. If you specify a `nodeCount` greater than `3`, {product-title} creates three Elasticsearch nodes that are Master-eligible nodes, with the master, client, and data roles. The additional Elasticsearch nodes are created as Data-only nodes, using client and data roles. control plane nodes (also known as the master nodes) perform cluster-wide actions such as creating or deleting an index, shard allocation, and tracking nodes. Data nodes hold the shards and perform data-related operations such as CRUD, search, and aggregations. Data-related operations are I/O-, memory-, and CPU-intensive. It is important to monitor these resources and to add more Data nodes if the current nodes are overloaded.
```

Comment:
```suggestion
The maximum number of Elasticsearch control plane nodes (also known as the master nodes) is three. If you specify a `nodeCount` greater than `3`, {product-title} creates three Elasticsearch nodes that are Master-eligible nodes, with the master, client, and data roles. The additional Elasticsearch nodes are created as Data-only nodes, using client and data roles. Control plane nodes (also known as the master nodes) perform cluster-wide actions such as creating or deleting an index, shard allocation, and tracking nodes. Data nodes hold the shards and perform data-related operations such as CRUD, search, and aggregations. Data-related operations are I/O-, memory-, and CPU-intensive. It is important to monitor these resources and to add more Data nodes if the current nodes are overloaded.
```

Reply:
@rolfedh Any thoughts on proper wording here? What is Elasticsearch doing in terms of conscious language?

_Elasticsearch control plane nodes (also known as the master nodes) is three_

Reply:
@mburke5678 The latest Elasticsearch software and documentation show no sign of addressing this issue. They still use the term, `master`. The parenthetical comment, above, seems to follow our guidelines.